howso/direct/core.py

```diff
@@ -1503,6 +1505,13 @@ def batch_react(
             index is the original 0-based index of the case as it was trained
             into the session. If this case does not exist, discriminative react
             outputs null, generative react ignores it.
+        post_process_features : iterable of str, optional
+            List of feature names that will be made available during the
+            execution of post_process feature attributes.
+        post_process_values : list of list of object or DataFrame, optional
```

Comment:
Docstring says `[[object]] | DataFrame | None` but typehint in signature says `[[object]] | None`. Which is it?

Reply:
ah good catch. Should just be `[[object]] | None`, no dataframes allowed at this point :).